sechub-shared-kernel/src/test/java/com/mercedesbenz/sechub/sharedkernel/util/FileChecksumSHA256ServiceTest.java

```diff
@@ -3,6 +3,11 @@
 
```

Comment:
Please rename this file, because this class works on streams not on files anymore.